dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/Utils.kt

```diff
@@ -20,3 +22,27 @@ fun SectionConfigBuilder.addStandardCellWithValueGetterFactory(
         valueGetter = valueGetter,
     )
 }
+
```

Comment:
So this is the generalized function to assemble the respective upload components, correct?

Reply:
Don't know what you mean, but this function is responsible for for adding a field-object into the UploadConfig.ts under some sub-category.

Reply:
Yeah that was what i meant.